fawltydeps/cli_parser.py

```diff
@@ -164,6 +164,14 @@ def populate_parser_paths_options(parser: argparse._ActionsContainer) -> None:
             " installed."
         ),
     )
+    parser.add_argument(
+        "--mapping",
+        type=Path,
+        metavar="PATH",
+        help=(
+            "Mapping of dependencies to imports defined by the user in form of toml file."
+        ),
+    )
```

Comment:
I do not think that users will want to write such complicated commands in the CLI. I think that if the custom mapping is be used it will be either via pyptoject.toml or via a custom mapping file.

I agree that maybe `--mapping` is too welcomy :D This is not an option that we want to push forward too much, as this is a power-user feature. Changed to `--custom-mapping-file`

It is a good idea for the parsing mapping from `pyproject.toml`. Added to [#290](https://github.com/tweag/FawltyDeps/issues/290).